### PR TITLE
Destination snowflake 1s1t: it works now

### PR DESCRIPTION
**Pull request by edgao**
https://github.com/airbytehq/airbyte/pull/29188 needs to merge first, to fix the `incrementalDedupDefaultNamespace` test. All other tests are passing.

T+D expected records have non-UTC timezone in airbyte_extracted_at because of https://github.com/airbytehq/airbyte/issues/29199. This also happens with old, non-1s1t syncs.

This PR is a grabbag of bugfixes:
* snowflake schema change detection is now implemented correctly (and has tests in BaseSqlGeneratorIntegrationTest)
* Added some more CDC tests + fixed a bug in snowflake's deleted_at null check
* GeneralStagingFunctions correctly finds the relevant StreamConfig
* `check` returns failure for 1s1t + gcs/s3 staging
* in 1s1t mode, wrap a bunch of raw table stuff in quotes to force lowercase table/stage/schema names
* Properly implement `AbstractSnowflakeTypingDedupingTest#dumpFinalTable`
* Fix timestamp formats in snowflake expected record files
* add support for `+/-08` and `+/-0800` timezone offsets in snowflake
* add support for raw schema overrides (along with [GSM secret](https://console.cloud.google.com/security/secret-manager/secret/SECRET_DESTINATION_SNOWFLAKE_1S1T_INTERNAL_STAGING_CREDS_RAW_SCHEMA_OVERRIDE/versions?project=dataline-integration-testing) for the new test)

I ran a faker -> snowflake sync locally. Try out `select * from "edgao_1s1t"."users" limit 10;` and `select * from "edgao_default_raw_schema"."edgao_1s1t_raw__stream_users" limit 10;` on our test instance!

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  destination-bigquery test report (commit [`a6365f26dc`](https://github.com/airbytehq/airbyte/pull/29174/commits/a6365f26dc66258b410d112b7130cd075d7da76e)) - ✅

⏲️  Total pipeline duration: 34mn43s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-bigquery/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization:dev                                                 | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-bigquery:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_1s1t_snowflake_impl2/1691447609/a6365f26dc66258b410d112b7130cd075d7da76e/destination-bigquery/1.7.5/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:a6365f26dc66258b410d112b7130cd075d7da76e)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-snowflake test report (commit [`a6365f26dc`](https://github.com/airbytehq/airbyte/pull/29174/commits/a6365f26dc66258b410d112b7130cd075d7da76e)) - ❌

⏲️  Total pipeline duration: 43mn16s 

| Step                                                                             | Result   |
|:---------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-snowflake/metadata.yaml     | ✅        |
| Connector version semver check                                                   | ✅        |
| Connector version increment check                                                | ✅        |
| QA checks                                                                        | ✅        |
| Build connector tar                                                              | ✅        |
| Build destination-snowflake docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization-snowflake:dev                                        | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-snowflake:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_1s1t_snowflake_impl2/1691447609/a6365f26dc66258b410d112b7130cd075d7da76e/destination-snowflake/1.3.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:a6365f26dc66258b410d112b7130cd075d7da76e)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-bigquery test report (commit [`2784fc370f`](https://github.com/airbytehq/airbyte/pull/29174/commits/2784fc370fae74cf8d3fc25b12073d991b07ae8b)) - ✅

⏲️  Total pipeline duration: 04mn08s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-bigquery/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization:dev                                                 | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-bigquery:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_1s1t_snowflake_impl2/1691452428/2784fc370fae74cf8d3fc25b12073d991b07ae8b/destination-bigquery/1.7.5/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:2784fc370fae74cf8d3fc25b12073d991b07ae8b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

File: airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json [hunk 1]
```diff
@@ -385,6 +385,18 @@
...
+        "type": "boolean",
+        "description": "(Beta) Use <a href=\"https://github.com/airbytehq/airbyte/issues/26028\" target=\"_blank\">Destinations V2</a>. Contact Airbyte Support to participate in the beta program.",
+        "title": "Use Destinations V2 (Early Access)",
+        "order": 10
+      },
+      "raw_data_schema": {
+        "type": "string",
+        "description": "(Beta) The schema to write raw tables into",
+        "title": "Destinations V2 Raw Table Schema (Early Access)",
+        "order": 11
```
Comment (evantahler) [inline airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json:399]: cc @Hesperide - we are going right into early-access for all users, no private beta thi time around

**Comment by octavia-squidington-iii** [comment]
##  destination-snowflake test report (commit [`614d197f58`](https://github.com/airbytehq/airbyte/pull/29174/commits/614d197f586ba52c16c21514ff12104ad65cc24b)) - ❌

⏲️  Total pipeline duration: 46mn15s 

| Step                                                                             | Result   |
|:---------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-snowflake/metadata.yaml     | ✅        |
| Connector version semver check                                                   | ✅        |
| Connector version increment check                                                | ✅        |
| QA checks                                                                        | ✅        |
| Build connector tar                                                              | ✅        |
| Build destination-snowflake docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization-snowflake:dev                                        | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-snowflake:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_1s1t_snowflake_impl2/1691453261/614d197f586ba52c16c21514ff12104ad65cc24b/destination-snowflake/1.3.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:614d197f586ba52c16c21514ff12104ad65cc24b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-bigquery test report (commit [`614d197f58`](https://github.com/airbytehq/airbyte/pull/29174/commits/614d197f586ba52c16c21514ff12104ad65cc24b)) - ✅

⏲️  Total pipeline duration: 33mn17s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-bigquery/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization:dev                                                 | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-bigquery:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_1s1t_snowflake_impl2/1691453261/614d197f586ba52c16c21514ff12104ad65cc24b/destination-bigquery/1.7.5/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:614d197f586ba52c16c21514ff12104ad65cc24b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

File: airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingSqlOperations.java [hunk 2]
```diff
@@ -30,24 +28,52 @@ public class SnowflakeInternalStagingSqlOperations extends SnowflakeSqlStagingOp
...
     this.nameTransformer = nameTransformer;
+    this.use1s1t = TypingAndDedupingFlag.isDestinationV2();
   }
 
   @Override
   public String getStageName(final String namespace, final String streamName) {
-    return nameTransformer.applyDefaultCase(String.join(".",
-        nameTransformer.convertStreamName(namespace),
-        nameTransformer.convertStreamName(streamName)));
+    if (use1s1t) {
```
Comment (jbfbell) [inline airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingSqlOperations.java:68]: Is something that only applies to 1s1t?

File: airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeSqlGenerator.java [hunk 3]
```diff
@@ -28,13 +28,13 @@ public class SnowflakeSqlGenerator implements SqlGenerator<SnowflakeTableDefinit
...
-  public StreamId buildStreamId(String namespace, String name, String rawNamespaceOverride) {
+  public StreamId buildStreamId(final String namespace, final String name, final String rawNamespaceOverride) {
     // TODO
     return new StreamId(namespace, name, rawNamespaceOverride, StreamId.concatenateRawTableName(namespace, name), namespace, name);
   }
 
   @Override
-  public ColumnId buildColumnId(String name) {
+  public ColumnId buildColumnId(final String name) {
     // TODO
```
Comment (jbfbell) [inline airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeSqlGenerator.java:38]: what's TODO?

File: airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeSqlGenerator.java [hunk 4]
```diff
@@ -174,14 +174,28 @@ private String extractAndCast(final ColumnId column, final AirbyteType airbyteTy
       return new StringSubstitutor(Map.of("column_name", column.originalName())).replace(
           """
           CASE
-            WHEN NOT ("_airbyte_data":"${column_name}"::TEXT REGEXP '\\\\d{1,2}:\\\\d{2}:\\\\d{2}(\\\\.\\\\d+)?(Z|[+\\\\-]\\\\d{1,2}:\\\\d{2})')
+            WHEN NOT ("_airbyte_data":"${column_name}"::TEXT REGEXP '\\\\d{1,2}:\\\\d{2}:\\\\d{2}(\\\\.\\\\d+)?(Z|[+\\\\-]\\\\d{1,2}(:?\\\\d{2})?)')
```
Comment (jbfbell) [inline airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeSqlGenerator.java:177]: nit: whenever I have a gnarly regex like this I like to add a comment with an example or two of what it is supposed to match

**Review by jbfbell — APPROVED** [review]

File: airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingSqlOperations.java (hunk 2 above)
Comment (edgao) [reply airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingSqlOperations.java:68]: yep. this prevents snowflake from uppercasing the schema/table name, which is a breaking change relative to current behavior.

File: airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeSqlGenerator.java (hunk 3 above)
Comment (edgao) [reply airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeSqlGenerator.java:38]: nothing 🤦 I'm sure we'll get some beta users reporting weird character handling, but in my current testing this seems to work fine. I'll remove the comments.

File: airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeSqlGenerator.java (hunk 4 above)
Comment (edgao) [reply airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeSqlGenerator.java:182]: good call, added comments for this

**Comment by octavia-squidington-iii** [comment]
##  destination-snowflake test report (commit [`9dfe2cb3b1`](https://github.com/airbytehq/airbyte/pull/29174/commits/9dfe2cb3b17b4f7e027d3a274e3016b715a54474)) - ❌

⏲️  Total pipeline duration: 230mn06s 

| Step                                                                             | Result   |
|:---------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-snowflake/metadata.yaml     | ✅        |
| Connector version semver check                                                   | ✅        |
| Connector version increment check                                                | ✅        |
| QA checks                                                                        | ✅        |
| Build connector tar                                                              | ✅        |
| Build destination-snowflake docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization-snowflake:dev                                        | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-snowflake:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_1s1t_snowflake_impl2/1691521003/9dfe2cb3b17b4f7e027d3a274e3016b715a54474/destination-snowflake/1.3.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:9dfe2cb3b17b4f7e027d3a274e3016b715a54474)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-bigquery test report (commit [`9dfe2cb3b1`](https://github.com/airbytehq/airbyte/pull/29174/commits/9dfe2cb3b17b4f7e027d3a274e3016b715a54474)) - ✅

⏲️  Total pipeline duration: 31mn46s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-bigquery/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization:dev                                                 | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-bigquery:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_1s1t_snowflake_impl2/1691521003/9dfe2cb3b17b4f7e027d3a274e3016b715a54474/destination-bigquery/1.7.5/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:9dfe2cb3b17b4f7e027d3a274e3016b715a54474)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-bigquery test report (commit [`ff77a79a0d`](https://github.com/airbytehq/airbyte/pull/29174/commits/ff77a79a0d88851e24b4a3d2b346c8e8012b0969)) - ✅

⏲️  Total pipeline duration: 32mn41s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-bigquery/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization:dev                                                 | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-bigquery:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_1s1t_snowflake_impl2/1691536852/ff77a79a0d88851e24b4a3d2b346c8e8012b0969/destination-bigquery/1.7.5/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:ff77a79a0d88851e24b4a3d2b346c8e8012b0969)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-teradata test report (commit [`ff77a79a0d`](https://github.com/airbytehq/airbyte/pull/29174/commits/ff77a79a0d88851e24b4a3d2b346c8e8012b0969)) - ❌

⏲️  Total pipeline duration: 14mn42s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-teradata/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| Connector version increment check                                               | ❌        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-teradata docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-teradata:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_1s1t_snowflake_impl2/1691536852/ff77a79a0d88851e24b4a3d2b346c8e8012b0969/destination-teradata/0.1.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:ff77a79a0d88851e24b4a3d2b346c8e8012b0969)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-teradata test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-snowflake test report (commit [`ff77a79a0d`](https://github.com/airbytehq/airbyte/pull/29174/commits/ff77a79a0d88851e24b4a3d2b346c8e8012b0969)) - ❌

⏲️  Total pipeline duration: 40mn28s 

| Step                                                                             | Result   |
|:---------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-snowflake/metadata.yaml     | ✅        |
| Connector version semver check                                                   | ✅        |
| Connector version increment check                                                | ✅        |
| QA checks                                                                        | ✅        |
| Build connector tar                                                              | ✅        |
| Build destination-snowflake docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization-snowflake:dev                                        | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-snowflake:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_1s1t_snowflake_impl2/1691536852/ff77a79a0d88851e24b4a3d2b346c8e8012b0969/destination-snowflake/1.3.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:ff77a79a0d88851e24b4a3d2b346c8e8012b0969)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-bigquery test report (commit [`dec948ac9f`](https://github.com/airbytehq/airbyte/pull/29174/commits/dec948ac9faf519a34d0889d7a97e80afad80116)) - ✅

⏲️  Total pipeline duration: 32mn46s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-bigquery/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization:dev                                                 | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-bigquery:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_1s1t_snowflake_impl2/1691544143/dec948ac9faf519a34d0889d7a97e80afad80116/destination-bigquery/1.7.5/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:dec948ac9faf519a34d0889d7a97e80afad80116)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-teradata test report (commit [`dec948ac9f`](https://github.com/airbytehq/airbyte/pull/29174/commits/dec948ac9faf519a34d0889d7a97e80afad80116)) - ❌

⏲️  Total pipeline duration: 14mn38s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-teradata/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| Connector version increment check                                               | ❌        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-teradata docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-teradata:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_1s1t_snowflake_impl2/1691544143/dec948ac9faf519a34d0889d7a97e80afad80116/destination-teradata/0.1.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:dec948ac9faf519a34d0889d7a97e80afad80116)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-teradata test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-snowflake test report (commit [`dec948ac9f`](https://github.com/airbytehq/airbyte/pull/29174/commits/dec948ac9faf519a34d0889d7a97e80afad80116)) - ✅

⏲️  Total pipeline duration: 39mn29s 

| Step                                                                             | Result   |
|:---------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-snowflake/metadata.yaml     | ✅        |
| Connector version semver check                                                   | ✅        |
| Connector version increment check                                                | ✅        |
| QA checks                                                                        | ✅        |
| Build connector tar                                                              | ✅        |
| Build destination-snowflake docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization-snowflake:dev                                        | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-snowflake:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_1s1t_snowflake_impl2/1691544143/dec948ac9faf519a34d0889d7a97e80afad80116/destination-snowflake/1.3.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:dec948ac9faf519a34d0889d7a97e80afad80116)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Pull request merged**